src/Component/Header/style.scss

```
@@ -0,0 +1,214 @@
+@import "../../style/mixin";
+@import "../../style/variables";
+.header {
+    height: 6rem;
+    background-color: $whiteColor;
+    box-shadow: 0 4px 20px rgb(38 38 38 / 20%);
+    position: fixed!important;
+    width: 100%;
+    z-index: 999;
+    top: 0;
+    margin-bottom: 32px;
+
+    &_btn-toggle {
+        display: none;
+        font-size: 2rem;
+        line-height: 1.5;
+        margin-left: 1rem;
+        
+        @include Responsive(Mobile) {
+            display: block ;
+            z-index: 100;
+        }
+    }
+
+    .container {
+        height: 100%;
+    }
+
+    &_list {
+        display: flex;
+        justify-content: space-between;
+        height: 100%;
+        align-items: center;
+    }
+
+    &_logo {
+        display: inline-block;
+        font-size: 1.75rem;
+        @include paddingY(0.3125rem,0.3125rem);
```

```suggestion
        @include paddingY(0.3125rem,0.3125rem);
        display: inline-block;
        font-size: 1.75rem;      
```
Đưa lên đầu cho dễ quan sát nhé, quên hết chữ ở PR1 rồi :D